package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@json-editor/json-editor",
+  "name": "@mia-platform/json-editor",
```

Comment:
On `master` we keep `json-editor`.
Before releasing our version, we'll make a branch where we set `mia-platorm`

Reply:
```suggestion
  "name": "@json-editor/json-editor",
```